File: java/soma_matrizes.java
import java.util.Locale;
import java.util.Scanner;

public class soma_matrizes {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int M, N;
		
		System.out.print("Quantas linhas vai ter cada matriz? ");
		M = sc.nextInt();
		System.out.print("Quantas colunas vai ter cada matriz?");
		N = sc.nextInt();
		
		int[][] matrizA = new int[M][N];
		int[][] matrizB = new int[M][N];
		int[][] matrizC = new int[M][N];
		
		System.out.println("Digite os valores da matriz A: ");
		for (int i = 0; i < M; i++) {
			for (int j = 0; j < N; j++) {
				System.out.print("Elemento [" + i + "," + j + "]: ");
				matrizA[i][j] = sc.nextInt();
			}
		}
		
		System.out.println("Digite os valores da matriz B: ");
		for (int i = 0; i < M; i++) {
			for (int j = 0; j < N; j++) {
				System.out.print("Elemento [" + i + "," + j + "]: ");
				matrizB[i][j] = sc.nextInt();
			}
		}
		
		System.out.println("MATRIZ SOMA: ");
		for (int i = 0; i < M; i++) {
			for (int j = 0; j < N; j++) {
				matrizC[i][j] = matrizA[i][j] + matrizB[i][j];
				System.out.print(matrizC[i][j] + " ");
			}
			System.out.println();
		}
		
		sc.close();
	}

}

File: c++/soma_matrizes.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {

    int m, n;

    cout << "Quantas linhas vai ter cada matriz? ";
    cin >> m;
    cout << "Quantas colunas vai ter cada matriz? ";
    cin >> n;

    int matrizA[m][n];
    int matrizB[m][n];
    int matrizC[m][n];

    cout << "Digite os valores da matriz A: " << endl;
    for(int i = 0; i < m; i++) {
        for(int j = 0; j < n; j++) {
            cout << "Elemento [" << i << "," << j << "]: ";
            cin >> matrizA[i][j];
        }
    }

    cout << "Digite os valores da matriz B: " << endl;
    for(int i = 0; i < m; i++) {
        for(int j = 0; j < n; j++) {
            cout << "Elemento [" << i << "," << j << "]: ";
            cin >> matrizB[i][j];
        }
    }

    cout << "MATRIZ SOMA: " << endl;
    for(int i = 0; i < m; i++) {
        for(int j = 0; j < n; j++) {
            matrizC[i][j] = matrizA[i][j] + matrizB[i][j];
            cout << matrizC[i][j] << " ";
        }
        cout << endl;
    }

    return 0;
}
